src/client/include/com/amazonaws/kinesis/video/client/Include.h

```diff
@@ -1140,6 +1142,63 @@ struct __StreamMetrics {
 
     // Last measured transfer rate in bytes per second
     UINT64 currentTransferRate;
+
+    // V1 metrics following
+
+    // Total stream duration - stream uptime
+    UINT64 uptime;
+
+    // Total transferred bytes
+    UINT64 transferredBytes;
+
+    // Total number of streaming sessions including new/errored/active
+    UINT64 totalSessions;

```

Comment:
I am wondering what the use case for total sessions would be?

Reply:
Not all of the metrics would be used with the same frequency or need. The total number of session is useful to compare against the total active session number to monitor whether we have sessions that are not streaming for some reason. In general, those numbers should be identical.